manifests/01-service-delete.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    release.openshift.io/delete: "true"
```

Comment:
The CVO will ignore this unless you set [profile annotations][1].  Checking your other manifests:

```console
$ git grep -oh 'include.release.openshift.io/.*' manifests | sort | uniq -c
     14 include.release.openshift.io/ibm-cloud-managed: "true"
     20 include.release.openshift.io/self-managed-high-availability: "true"
```

[1]: https://github.com/openshift/enhancements/blob/ce4d303db807622687159eb9d3248285a003fabb/enhancements/update/cluster-profiles.md

Reply:
Wow, our manifest is so old it predates all that stuff. Thanks for the quick catch. You saved us some QE time :1st_place_medal: